### PR TITLE
Bug 1134414 ff37 whatsnew page

### PR DESCRIPTION
**Pull request by alexgibson**
Marking this as **do not merge** for now as the exact target parts are still being worked on, but this should be good for code review.

Note: there are currently issues with basket-dev, so to test the newsletter whomever reviews this PR will likely need to set `BASKET_URL = 'http://basket.allizom.org'` in their `local.py`

### CODE REVIEW

**Comment by cargoworker** [comment]
Can one of the admins verify this patch?

File: bedrock/firefox/templates/firefox/whatsnew-fx37.html [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        {% set newsletter_submit = _('Send') %}
+        {% set newsletter_thankyou_head = _('Your link has been sent!') %}
+        {% set newsletter_thankyou_content = thank_you_message() %}
+
+        {{ email_newsletter_form(newsletter_id, title=newsletter_title, submit_text=newsletter_submit, thankyou_head=newsletter_thankyou_head,thankyou_content=newsletter_thankyou_content) }}
+        <footer>
+          {% trans url=settings.GOOGLE_PLAY_FIREFOX_LINK %}
+            Also available from <a href="{{ url }}">Google Play</a>.
+          {% endtrans %}
+          <br>{{ _('The intended recipient of the email must have consented.') }} <a href="#">{{ _('Learn more') }}</a>
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/whatsnew-fx37.html]: Note to self: still need a url here to replace the `#`

File: bedrock/firefox/templates/firefox/whatsnew-fx37.html [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+              <p>{{ thank_you_message() }}</p>
+            </div>
+            <div id="sms-spinner" class="hidden"></div>
+          </div>
+          <footer>
+            {% trans url=settings.GOOGLE_PLAY_FIREFOX_LINK %}
+              Also available from <a href="{{ url }}">Google Play</a>.
+            {% endtrans %}
+            <br>
+            {{ _('SMS service available in the U.S. only.') }} {{ _('SMS & data rates may apply. The intended recipient of the SMS must have consented.') }} <a href="#">{{ _('Learn more') }}</a>
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/whatsnew-fx37.html]: Note to self: still need a url here to replace the `#`

**Comment by alexgibson** [comment]
@jpetto @craigcook r?

**Comment by jpetto** [comment]
Everything looks top notch to me. r+ :basketball:

Do we need a more dedicated review of the Python bits?

**Comment by alexgibson** [comment]
Thanks @jpetto!

@jgmize - care to give @pmclanahan's python additions a quick review?

**Comment by jgmize** [comment]
The python bits lgtm, and since the frontend already got the r+ from @jpetto I'm going to go ahead and merge.

**Comment by alexgibson** [comment]
Thanks, @jgmize!

**Pull request merged**